### ISSUE TITLE
Cascading Context menu

### ISSUE THREAD

**Issue by buggyj** [buggyj/tiddlyclip#76]
### Discussed in https://github.com/buggyj/tiddlyclip/discussions/74

<div type='discussions-op-text'>

<sup>Originally posted by **arunnbabu81** March 16, 2022</sup>
It would be better if the context menu could be modified like this  - nested or cascading context menu ( don't know whether that's the right term or not)

![image](https://user-images.githubusercontent.com/67494083/158671847-0a90e91b-7e65-46be-b76b-f0df85733f04.png)
Is that a feasible option?

I have a huge list of clip rules, so such a nested context menu will be very useful

</div>

**Comment by arunnbabu81** [comment]
> ### Discussed in #74
> 
> Originally posted by **arunnbabu81** March 16, 2022 It would be better if the context menu could be modified like this - nested or cascading context menu ( don't know whether that's the right term or not)
> 
> ![image](https://user-images.githubusercontent.com/67494083/158671847-0a90e91b-7e65-46be-b76b-f0df85733f04.png) Is that a feasible option?
> 
> I have a huge list of clip rules, so such a nested context menu will be very useful

Just a reminder to consider this when you are updating the add on a later date.

**Comment by buggyj** [comment]
what would the nesting look like:

wiki->section->rules??

**Comment by arunnbabu81** [comment]
> what would the nesting look like:
> 
> tiddlyclip->wiki->section->rules??

Yes. Thats what I was thinking of. Is there any other way to do it?

**Comment by buggyj** [comment]
At present there is a focused wiki and focused section. It would be possible to keep the focused wiki...

I will need to re-write this part of tiddlyclip sometime this year as google are no longer going to support v2 addons so I will have to port tiddlyclip to v3.
